package.json

```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "build": "tsc",
     "test": "MOCHA_WEBDRIVER_STACKTRACES=1 mocha 'test/browser/**/*.{js,ts}'",
-    "test-debug": "MOCHA_WEBDRIVER_STACKTRACES=1 mocha --bail test/browser/**/*.{js,ts} --no-exit",
-    "test-manual": "webpack-dev-server --config test/fixtures/webpack.config.js",
+    "test-debug": "NODE_OPTIONS=--openssl-legacy-provider MOCHA_WEBDRIVER_STACKTRACES=1 mocha --bail test/browser/**/*.{js,ts} --no-exit",
```

Comment:
I wonder if upgrading Webpack will resolve this. Anyway, ok to worry about later since it's a separate issue.